### PR TITLE
feat(web): Project page news

### PR DESCRIPTION
**Pull request by RunarVestmann**
# Project page news

## What

* Add news list screen for project pages
* Add news details screen for project pages
* Restructure the setup in the code by adding a NewsList component that can be reused for both organization- and project pages
* Removed the NewsItem screen and replaced it with two screens (one for organization pages and another for projedct pages) that wrap around the NewsArticle component.
* Changed namespace props types since they are outdated (we are JSON.parsing the namespace fields so the correct type should be Record<string, string>)
* Changed props for LatestNewsSlice so it can be used by both project pages and organization pages
* Renamed OrganizationSlice to SliceMachine (other suggestions are welcome) since slices are used by both project pages and organization pages
* Added bottom slices field to Project pages since we want to reuse the LatestNewsSlice component for both organization pages and project pages
* Add id to the graph card model since we are getting a lint error when using a slice union's id as a React key
* Make linkGroup model primarylink field nullable (since it can be null and causes a 500 error if it is)

## Why

* We were setting up a new project page and quickly realised that we are missing some required features that are already present on organization pages (being able to have news and support for more slices, like the new form slice and so forth).

### NewsList view for project pages added
Screenshot 2022-09-08 at 10 06 10 2

### NewsArticle view for project pages added
Screenshot 2022-09-08 at 10 13 11 2

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/web/screens/Project/ProjectNewsList.tsx [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+    },
+  ]
+
+  const baseRouterPath = router.asPath.split('?')[0].split('#')[0]
+
+  const currentNavItem = projectPage.sidebarLinks.find(
+    ({ primaryLink }) => primaryLink.url === baseRouterPath,
+  )?.primaryLink
+
+  console.log(currentNavItem)
```
Comment (kksteini) [inline apps/web/screens/Project/ProjectNewsList.tsx:93]: :eyes:

Comment (RunarVestmann) [reply apps/web/screens/Project/ProjectNewsList.tsx:93]: aah, I missed a spot :)

**Review by kksteini — APPROVED** [review]

**Review by kksteini — APPROVED** [review]

File: apps/web/screens/Project/ProjectNewsList.tsx (hunk 1 above)
Comment (RunarVestmann) [reply apps/web/screens/Project/ProjectNewsList.tsx]: I've removed the console.log

**Review by Ingimarsson — APPROVED** [review]
Removed automerge and approved, just needs a `<title>` on the project news list.

**Pull request merged**